README.md

```diff
@@ -254,11 +247,11 @@ Deploy the application to Minikube using the Marblerun.
     1. Bring the Coordinator into recovery mode
```

Comment:
Maybe add some information here about the certificate warning one will receive, since we no longer manually add the certificate to the browser